app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+from rest_framework import serializers
+
+from experimenter.experiments.models import (
+    NimbusBranch,
+    NimbusBucketRange,
+    NimbusExperiment,
+)
+
+
+class NimbusBucketRangeSerializer(serializers.ModelSerializer):
+    randomizationUnit = serializers.ReadOnlyField(
+        source="isolation_group.randomization_unit"
+    )
+    namespace = serializers.ReadOnlyField(source="isolation_group.namespace")
+    total = serializers.ReadOnlyField(source="isolation_group.total")
+
+    class Meta:
+        model = NimbusBucketRange
+        fields = (
+            "randomizationUnit",
+            "namespace",
+            "start",
+            "count",
+            "total",
+        )
+
+
+class NimbusBranchSerializer(serializers.ModelSerializer):
+    feature = serializers.SerializerMethodField()
+
+    class Meta:
+        model = NimbusBranch
+        fields = ("slug", "ratio", "feature")
+
+    def get_feature(self, obj):
+        return {
+            "featureId": obj.experiment.feature_config.slug,
+            "enabled": obj.enabled,
+            "value": obj.value,
+        }
+
+
+class NimbusExperimentSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField(source="slug")
+    userFacingName = serializers.ReadOnlyField(source="name")
+    userFacingDescription = serializers.ReadOnlyField(source="public_description")
+    isEnrollmentPaused = serializers.ReadOnlyField(source="is_paused")
+    bucketConfig = NimbusBucketRangeSerializer(source="bucket_range")
+    probeSets = serializers.SerializerMethodField()
+    branches = NimbusBranchSerializer(many=True)
+    targeting = serializers.SerializerMethodField()
+    startDate = serializers.ReadOnlyField(default=None)
+    endDate = serializers.ReadOnlyField(default=None)
+    proposedDuration = serializers.ReadOnlyField(source="proposed_duration")
+    proposedEnrollment = serializers.ReadOnlyField(source="proposed_enrollment")
+    referenceBranch = serializers.SerializerMethodField()
+
+    class Meta:
+        model = NimbusExperiment
+        fields = (
+            "slug",
+            "id",
+            "application",
+            "userFacingName",
+            "userFacingDescription",
+            "isEnrollmentPaused",
+            "bucketConfig",
+            "probeSets",
+            "branches",
+            "targeting",
+            "startDate",
+            "endDate",
+            "proposedDuration",
+            "proposedEnrollment",
+            "referenceBranch",
+        )
+
+    def get_probeSets(self, obj):
+        return list(obj.probe_sets.all().order_by("slug").values_list("slug", flat=True))
+
+    def get_referenceBranch(self, obj):
```

Comment:
Why not just call this the controlBranch?

Reply:
The schema is defined here:

https://github.com/mozilla/nimbus-shared/blob/main/types/experiments.ts

Reply:
Ok, perhaps obvious follow-up question, why not call this Reference Branch everywhere? (In our UX, in the model, etc.)

Reply:
Could be, but thinking about how things have gone in the past, the external facing names can change frequently and by that time it could wind up being major refactors to make the code reflect the external facing names, which we have done, but becomes more and more difficult over time.  This is how we wound up with `variants` vs `branches` for instance.  So we could take the opportunity to rename it in the code now while Nimbus is still young, but that doesn't guarantee it won't change again in the future.